.github/workflows/run-compile-requirements.yml

```diff
@@ -0,0 +1,32 @@
+name: Generate requirement files
+
+on:
+  push:
+    branches: [requirements]
+
+  pull_request:
+    branches: [requirements]
+
+jobs:
+  call-compile-requirements-linux:
+    uses: ./.github/workflows/compile-requirements.yml
+    with:
+      requirements_files: '["base.in dev.in gui-dev.in gui.in linters.in"]'
+      os: '["ubuntu-latest"]'
+      python: '["3.6", "3.7", "3.8", "3.9"]'
```

Comment:
do we need all the python versions only for linux?  I would think we could just do 3.9 only.

Reply:
That's a good point. I am currently mimicking what was in the original test runs (currently all these versions of Python in Linux are tested, as well as the two other tests on Mac and Windows). We should revisit whether we need more tests (e.g. different python versions on Windows/Mac) or less tests (e.g. only one version on Linux).

Perhaps this can be informed by more telemetry (e.g. [bug 1767534](https://bugzilla.mozilla.org/show_bug.cgi?id=1767534)).